src/DynamoCore/DynamoCore.csproj

```diff
@@ -149,7 +149,9 @@
     <Copy SourceFiles="$(SolutionDir)..\README.md" DestinationFiles="$(OutputPath)README.txt" />
     <Copy SourceFiles="$(SolutionDir)..\doc\distrib\License.rtf" DestinationFolder="$(OutputPath)" />
     <Copy SourceFiles="$(SolutionDir)..\doc\distrib\TermsOfUse.rtf" DestinationFolder="$(OutputPath)" />
+    <!--TODO: Remove in Dynamo 3.0-->
```

Comment:
Why not remove it now and add it to `legacy_remove_me` so we remember we have to remove this later?